WordPress/src/main/res/values/strings.xml

```diff
@@ -1166,6 +1166,7 @@
     <string name="reader_empty_posts_request_failed">Unable to retrieve posts</string>
     <string name="reader_empty_posts_in_tag">No posts with this tag</string>
     <string name="reader_empty_posts_in_tag_updating">Fetching posts…</string>
+    <string name="reader_empty_posts_in_custom_list">The sites in this list haven\'t posted anything recently</string>
```

Comment:
Should we use the same exact wording of the web version? `The sites in this list have not posted anything recently.`